auto_circuit/tasks.py

```diff
@@ -205,6 +205,8 @@ def init_task(self):
             bs_2 = bs[1] if isinstance(bs, tuple) else bs
             count_2 = b_count[1] if isinstance(b_count, tuple) else b_count
             has_tokenizer = hasattr(model, "tokenizer") and model.tokenizer is not None
+            if has_tokenizer and hasattr(model.tokenizer, "add_bos_token"):
+                model.tokenizer.add_bos_token = False
```

Comment:
Can you explain the reasoning behind this?

Reply:
(sorry should have made an issue) failing to set `model.tokenizer.add_bos_token=False` for pythia models causes the bos token to be added to answers and wrong answers. 

To reproduce:

```python
from auto_circuit.tasks import CAPITAL_CITIES_PYTHIA_70M_AUTOENCODER_COMPONENT_CIRCUIT_TASK
task = CAPITAL_CITIES_PYTHIA_70M_AUTOENCODER_COMPONENT_CIRCUIT_TASK.init_task()
batch = next(iter(task.train_loader))
print(batch.answers[0]) # tensor([[    0, 39759]], device='cuda:0')
```

Reply:
Okay thanks, can you add a comment explaining this?